consent-ws/src/main/java/org/broadinstitute/consent/http/mail/freemarker/DatasetOwnerListModel.java

```diff
@@ -9,7 +11,7 @@
     public DatasetOwnerListModel(String ownerName, String datasetName, String datasetId) {
         this.ownerName = ownerName;
         this.datasetName = datasetName;
-        this.datasetId = datasetId;
+        this.datasetId = StringUtils.isEmpty(datasetId) ? "--" : datasetId;
```

Comment:
if the dataset is not empty are we expecting this to be an integer? is this the same value as the dataSetId used in the association?

Reply:
Actually this is a wrong variable name, it should be renamed objectId, which is a string type. This value can be empty, so we replace it with "-- " on the e-mail template. 
I'll make this change.